test/features/comments.feature

```diff
@@ -79,29 +79,102 @@ Feature: Comments
+        Then I submit a comment with subject "Test subject" and comment "This is a test comment"
+        Then I take a screenshot
+        Then I wait for 3 seconds
+        Then I should see an element with xpath "//div[contains(@class, 'comment-wrapper highlight') and contains(string(), 'This is a test comment')]"
+
+    @comment-add @comment-email
+    Scenario: When a logged-in user submits a comment on a Data Request the email should contain title and comment
+        Given "CKANUser" as the persona
+        When I log in
+        And I go to data request "Test Request" comments
+        Then I should see an element with xpath "//h3[contains(string(), 'Add a comment')]"
+        Then I submit a comment with subject "Test Request" and comment "This is a test data request comment"
+        Then I take a screenshot
+        When I wait for 5 seconds
+        Then I should receive a base64 email at "dr_admin@localhost" containing "Dataset title: Test Request"
+        And I should receive a base64 email at "dr_admin@localhost" containing "Comment: This is a test data request comment"
+
+    @comment-delete
+    Scenario: When an Org Admin visits a dataset belonging to their organisation, they can delete a comment and the comment will be marked with delete by username.
+        Given "TestOrgAdmin" as the persona
+        When I log in
+        Then I go to dataset "warandpeace" comments
         And I press the element with xpath "//a[@title='Delete comment']"
         Then I should see "Are you sure you want to delete this comment?" within 1 seconds
         Then I press the element with xpath "//button[contains(string(), 'Confirm')]"
         Then I take a screenshot
-        Then I should see "This comment was deleted." within 2 seconds
+        Then I should see "This comment was deleted by test_org_admin." within 2 seconds
 
     @comment-delete
-    Scenario: When an Org Admin visits a data request belonging to their organisation, they can delete a comment and the comment will be marked as deleted.
+    Scenario: When an Org Admin visits a data request belonging to their organisation, they can delete a comment and the comment will be marked with delete by username.
         Given "Admin" as the persona
         When I log in
         And I go to data request "Test Request" comments
         And I press the element with xpath "//a[@title='Delete comment']"
         Then I should see "Are you sure you want to delete this comment?" within 1 seconds
         Then I press the element with xpath "//button[contains(string(), 'Confirm')]"
         Then I take a screenshot
-        Then I should see "This comment was deleted." within 2 seconds
+        Then I should see "This comment was deleted by admin." within 2 seconds
+
+    @comment-tab
+    Scenario: Non-logged in users should not see comment form in dataset tab
+        Given "Unathenticated" as the persona
```

Comment:
Misspelled "Unauthenticated". Possibly this is misspelled in the original definition?

Reply:
@ThrawnCA have addressed via https://github.com/MarkCalvert/ckanext-ytp-comments/commit/036112d3f95d8b4f99b9471c3c6c0d42903dd04c